hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNode.java

```diff
@@ -1015,6 +1015,10 @@ private void startCommonServices(Configuration conf) throws IOException {
       LOG.info(getRole() + " service RPC up at: "
           + rpcServer.getServiceRpcAddress());
     }
+    if (rpcServer.getLifelineRpcAddress() != null) {
+      LOG.info(getRole() + " lifeline RPC up at: "
```

Comment:
@haiyang1987 Thank you for your contribution! For newly added logs, should we use {} instead of splicing?

```
LOG.info("{} lifeline RPC up at: {}.", getRole(), rpcServer.getLifelineRpcAddress());
```

Reply:
Thanks @slfan1989 for your suggestion, i will update it later.